styles.css

```diff
@@ -593,6 +798,262 @@ h3 {
     height: 140vh;
   }
 
+  /* modal for destop */
+  .modal-container {
+    overflow-x: scroll;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    overflow: auto !important;
+    background-color: #c1c7d0;
+    z-index: 9999;
+    width: 90%;
+    height: 80%;
+    padding: 100px 90px 60px;
+  }
+
```

Comment:
- [ ] Please make sure that the popup window is responsive 

![image](https://user-images.githubusercontent.com/74744787/162446421-af3ced65-5f2a-4c9f-9428-c1614b2b9018.png)

![image](https://user-images.githubusercontent.com/74744787/162446851-fa3096d4-546b-4110-8d36-be4c359ccd90.png)


please fix this weird look of the close button it supoosed to look exactly as in the Figma design 

![image](https://user-images.githubusercontent.com/74744787/162447461-ebc22b58-75ce-4ec2-a65a-57832da55185.png)